apps/src/code-studio/pd/workshop_enrollment/enroll_form.jsx

```diff
@@ -113,6 +147,25 @@ export default class EnrollForm extends React.Component {
     return roleWithDescription;
   }
 
+  csfCoursesPlanned() {
+    if (!this.state.csf_courses_planned) {
+      return undefined;
+    }
+    const processedCourses = [];
+    this.state.csf_courses_planned.forEach((g) => {
+      if (g === `${OTHER} ${EXPLAIN}`) {
```

Comment:
What is this magic `===` ?

Reply:
The triple equals checks that both value and type are the same. Potentially overkill in this particular block but it's consistent with the surrounding code and is safer anyway.

Reply:
Our linter requires it in many scenarios, although I'm not sure exactly what the rule is